src/services/mongoose/users.ts

```diff
@@ -8,99 +8,122 @@ import {
 import { User } from "@services/mongoose/schema"
 import { onboardingEarn } from "@config/app"
 
-export const caseInsensitiveRegex = (input) => {
+export const caseInsensitiveRegex = (input: string) => {
   return new RegExp(`^${input}$`, "i")
 }
 
-export const UsersRepository = (): IUsersRepository => {
-  const findById = async (userId: UserId): Promise<User | RepositoryError> => {
+export const UsersRepository = () => {
+  // Remove when old GQL api is deprecated => only use domain user
+  const findByIdIncludeRaw = async (
```

Comment:
just leave the method `findById` and move the comment to line 27, but again, it breaks the purpose of clean up